src/Integration.Vsix.UnitTests/CFamily/IntegrationTests/CFamily_CLangAnalyzer_IntegrationTests.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.IO;
+using System.Threading;
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json;
+using SonarLint.VisualStudio.Integration.Vsix.CFamily;
+
+namespace SonarLint.VisualStudio.Integration.UnitTests.CFamily.IntegrationTests
+{
+    [TestClass]
+    public class CFamily_CLangAnalyzer_IntegrationTests
+    {
+        private string testsDataDirectory;
+
+        [TestInitialize]
+        public void TestInitialize()
+        {
+            testsDataDirectory = Path.Combine(Environment.CurrentDirectory, "CFamily\\IntegrationTests\\");
```

Comment:
Relying on `Environment.CurrentDirectory` is quite fragile and makes a lot of assumptions about the execution environment.

A less fragile option (i.e. one that makes fewer assumptions) is to look for the files relative to the current assembly  e.g.
```
            var resourcesPath = Path.Combine(
                Path.GetDirectoryName(typeof(MyTestClass).Assembly.Location), "TestResources");
``` 
(adapted from [here](https://github.com/SonarSource/sonarlint-visualstudio/blob/558296b44d059b55830e22a0553c8613f038d4a0/src/Core.UnitTests/CFamily/CFamilySonarWayRulesConfigProviderTests.cs#L92))

The most robust option is to embed the resources in the assembly and unpack them when needed, but that's more work so I'd only normally do that for production code.